File: regionalSwordFernDieOff/import/extract.R
library(XML)
file <- "doc.kml"
doc <- xmlParse(file)
str(getNodeSet(doc, "//Docfdsdfument"))

doc = xmlTreeParse(system.file("exampleData", "tagnames.xml", package = "XML"), useInternalNodes = TRUE)
length(getNodeSet(doc, "/doc//b[@status]"))
getNodeSet(doc, "/doc//b[@status='foo']")

File: regionalSwordFernDieOff/import/extract.py
# status: abandoned for now
from xml.etree import ElementTree as etree
filename = "doc.kml"
xmlDoc = etree.parse(filename)
len(xmlDoc.findall("Folder"))  # /Placemark"))
len(placeMarks)

from pykml import parser

with open(filename) as f:
    doc.parser.parse(f)
    
